index.html

```diff
@@ -59,6 +59,47 @@
         </picture>
     </div>
 </div>
+
+<h2 style="text-align: center; font-size: 1.4em;">Getting more done <span style="color: #b0197e;">after 2am</span> than most people do all day.</h2>
```

Comment:
Could this pad a bit more? It squishes a bit on mobile

Reply:
I'll try padding it a bit.

Reply:
Addressed in _sass/_typography.scss lines 36-45 of the diff. Changes applied to dev site.